CONTRIBUTING.md

```diff
@@ -218,6 +218,43 @@ or with the `runtests.jl` script, e.g. to run `test/bitarray.jl` and `test/math.
 
 Make sure that [Travis](http://www.travis-ci.org) greenlights the pull request with a [`Good to merge` message](http://blog.travis-ci.com/2012-09-04-pull-requests-just-got-even-more-awesome).
 
+#### Modifying base more efficiently with Revise.jl
+
+If you install the [Revise](https://github.com/timholy/Revise.jl)
```

Comment:
Of course the github link explains what `Rebase.jl` is doing, but maybe a short description here (tracks file changes and automatically updates function definitions) would clarify the logic behind the proposed procedure.